Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM docker.io/node:16.15.0-alpine
 ARG APP_ROOT=/opt/app-root/src
 ENV NO_UPDATE_NOTIFIER=true \
     APP_PORT=3000
-WORKDIR ${APP_ROOT}
 
-COPY . .
+COPY . ${APP_ROOT}
+WORKDIR ${APP_ROOT}/app
 RUN npm ci
```

Comment:
does this mean the docker image includes all the npm dependencies?

Reply:
This change copies the entire code repository into the docker image instead of just the app folder, excluding node_modules and other ignored files.